qhub/template/image/jupyterlab/environment.yaml

```diff
@@ -54,3 +54,4 @@ dependencies:
  - pip:
      # vscode jupyterlab launcher
      - git+https://github.com/betatim/vscode-binder
+     - git+https://github.com/dask/dask-labextension.git@main
```

Comment:
Should we rather install it from the commit hash?

Reply:
I would appreciate that as well, even though the labextension repo does not release frequently, I think it's worth pinning.

Reply:
Good idea! Done.